R/enframe.R

```diff
@@ -1,10 +1,10 @@
-#' Converting atomic vectors to data frames
+#' Converting atomic vectors to data frames, and vice versa
```

Comment:
Excitement about `deframe()` brought my eyeballs here.

`enframe()` works for lists. Should the "atomic" qualification be dropped or is that an undocumented feature you are not ready to explicitly support?